website/screens/components/date-input/code/examples/uncontrolled.ts

```diff
@@ -1,22 +1,42 @@
-import { DxcDateInput, DxcInset } from "@dxc-technology/halstack-react";
+import { DxcDateInput, DxcInset, DxcButton, DxcStack } from "@dxc-technology/halstack-react";
+import { useRef } from "react";
 
 const code = `() => {
+  const inputRef = useRef();
+
+  const handleSubmit = () => {
+    const input = inputRef.current.getElementsByTagName("input")[0];
+    console.log(input.value);
+  };
+
   return (
-    <DxcInset space="large">
-      <DxcDateInput
-        size="fillParent"
-        label="Uncontrolled"
-        helperText="Helper text"
-        defaultValue="10-08-1998"
-        clearable
-      />
-    </DxcInset>
+    <>
+      <DxcInset space="large">
```

Comment:
Update the returned JSX code to match the current uncontrolled example in other components:
```
  return (
    <DxcInset space="large">
      <DxcStack gutter="large" align="start">
        <DxcSelect
          label="Label"
          defaultValue="4"
          options={options}
          size="fillParent"
          ref={selectRef}
        />
        <DxcButton
          onClick={handleSubmit}
          label="Submit"
        />
      </DxcStack>
    </DxcInset>
  );
```